tracking_manager/models/models.py

```diff
@@ -120,7 +121,11 @@ def _tm_prepare_o2m_tracking(self):
             values = initial_values.setdefault(record.id, {})
             if values is not None:
                 for fname in fnames:
-                    values.setdefault(fname, record[fname])
+                    try:
+                        values.setdefault(fname, record[fname])
+                    except AccessError:
+                        # User does not have access to the field (example with groups)
+                        break
```

Comment:
I would say this for getting the rest of the fields:

```suggestion
                        continue
```